DTXManiaプロジェクト/コード/全体/ConfigIniToSoundGroupLevelControllerBinder.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using FDK;
+
+namespace DTXMania
+{
+    /// <summary>
+    /// The ConfigIniToSoundGroupLevelControllerBinder allows for updated sound
+    /// group level values, and keyboard sound level adjustment increment
+    /// values, to flow between CConfigIni and the SoundGroupLevelController
+    /// without either of those two classes being aware of one another.
+    /// See those classes properties, methods, and events for more details. 
+    /// </summary>
+    internal static class ConfigIniToSoundGroupLevelControllerBinder
```

Comment:
Here is the new ConfigIniToSoundGroupLevelControllerBinder. The doc comment and the code are well worth reading.